kegtab/build.gradle

```diff
@@ -44,7 +43,6 @@ repositories {
 
 android {
     compileSdkVersion 28
-    buildToolsVersion '25.0.3'
```

Comment:
Removed this because Android Studio was throwing this warning:

```
The specified Android SDK Build Tools version (25.0.3) is ignored, as it is below the minimum supported version (29.0.2) for Android Gradle Plugin 4.1.1.
Android SDK Build Tools 29.0.2 will be used.
To suppress this warning, remove "buildToolsVersion '25.0.3'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.
Remove Build Tools version and sync project
Affected Modules: kegtab
```